swim/ping_request_sender.go

```diff
@@ -112,6 +115,29 @@ func (p *pingRequestSender) MakeCall(ctx json.Context, res *pingResponse) <-chan
 	return errC
 }
 
+// indirectPing is used to check if a target node can be reached indirectly.
+// The indirectPing is performed by sending a specifiable amount of ping
+// requests nodes in n's membership.
+func indirectPing(n *Node, target string, amount int, timeout time.Duration) (reached bool, errs []error) {
```

Comment:
The go tour suggests otherwise: "These names should be used to document the meaning of the return values." https://tour.golang.org/basics/7

They indeed also add some other functionality, which I use a bit off here. (using errs).